docs/tutorials/basics/60_methods.md

```diff
@@ -163,3 +163,32 @@ puts life_universe_and_everything + 1 # Error: method top-level life_universe_an
 ```
 
 Now the compiler can show us exactly where the problem is originated. As we can see, providing type information is really useful for finding errors at compile time.
+
+## Blocks
+
+While working with *collections* (just as an example) we may have came across code that looks like this:
+
+```crystal-play
+["Hello", "World", 42].each do |elem|
+  puts elem
+end
+```
+
+or maybe something like this:
+
+```crystal-play
+arr = [1, 2, 3].map do |elem|
+  elem * 2
+end
+
+puts arr
+```
+
+In the first example we are using the method [Indexable#each](https://crystal-lang.org/api/Indexable.html#each%28%26%3AT-%3E%29-instance-method).
+And in the second example we are using the method [Enumerable#map](https://crystal-lang.org/api/Enumerable.html#map%28%26%3AT-%3EU%29%3AArray%28U%29forallU-instance-method).
+
+To both methods we are passing a *block of code* (delimited by the keywords `do` and `end`) as an argument.
+
+This *block of code*, commonly called *block* is a sequence of interactions, which will be executed at some point. When exactly? It depends on the method that receives the *block* as a parameter (in these examples `each` and `map`).
```

Comment:
Italics are good for the first occurrence of a topic. Subsequent occurrences should not be emphasized (in a section).

```suggestion
This block of code, commonly called *block* is a sequence of interactions, which will be executed at some point. When exactly? It depends on the method that receives the block as a parameter (in these examples `each` and `map`).
```